src/nectarchain/dqm/pixel_timeline.py

```diff
@@ -0,0 +1,155 @@
+from dqm_summary_processor import DQMSummary
+from matplotlib import pyplot as plt
+from ctapipe.visualization import CameraDisplay
+from ctapipe.instrument import CameraGeometry
+from ctapipe.coordinates import EngineeringCameraFrame
+import numpy as np
+
+
+class PixelTimeline_HighLowGain(DQMSummary):
```

Comment:
Class names should use CapWords convention

Reply:
done